pyproject.toml

```diff
@@ -18,13 +18,14 @@ license = {file="LICENSE"}
 dynamic = ["version"]
 
 # Dependencies
-requires-python = ">=3.11"
+requires-python = ">=3.9"
```

Comment:
To match benchcab, otherwise it won't install in the benchcab-dev environment. It doesn't seem to be an issue as we are not using any super bleeding-edge features of more modern versions.